src/plugins/vis_types/vislib/public/plugin.ts

```diff
@@ -87,11 +91,9 @@ export class VisTypeVislibPlugin
     core: CoreStart,
     { data, usageCollection, fieldFormats }: VisTypeVislibPluginStartDependencies
   ) {
+    setCoreStart(core);
```

Comment:
I decided to consolidate the getters to a single one, since doing so didn't seem to create a very big impact.

Reply:
I agree the duplicate work for each required core service is tedious. If it becomes an issue it would be nice to abstract this to `_.pick` only the required services, instead of multiple getters/setters.